pkg/minikube/problem/err_map.go

```diff
@@ -119,6 +124,11 @@ var vmProblems = map[string]match{
 		URL:    "http://mikko.repolainen.fi/documents/virtualization-with-kvm",
 		Issues: []int{2991},
 	},
+	"OLD_KVM_VERSION": {
+		Regexp: re(`Too old KVM version found`),
+		Advice: "too old KVM version. Jump forward to new valid KVM version as a workaround.",
```

Comment:
Add URL to documentation on how to upgrade.

Reply:
@josedonizetti done!

Reply:
done